code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -87,6 +87,7 @@
 	worn_icon_state = "cshotgun"
 	w_class = WEIGHT_CLASS_HUGE
 	semi_auto = TRUE
+	projectile_damage_multiplier = 1.4
```

Comment:
I'm only passing by, but I figured I'd highlight that this isn't actually mentioned in the body of your PR at all. Was this an intended change?